tracepoint_decode/src/lib.rs

```diff
@@ -15,8 +15,6 @@ pub use enumerator::EventHeaderEnumeratorError;
 pub use enumerator::EventHeaderEnumeratorState;
 pub use enumerator::EventHeaderEventInfo;
 pub use enumerator::EventHeaderItemInfo;
-pub use enumerator::NameAndTagDisplay;
-pub use enumerator::NameDisplay;
 
```

Comment:
Note: These types moved to the new `display` mod. No longer exported at crate level since they will usually be used without mentioning the type name.